include/nlohmann/detail/output/serializer.hpp

```diff
@@ -811,6 +811,18 @@ class serializer
         return state;
     }
 
+    number_unsigned_t remove_sign(number_unsigned_t x)
+    {
+        assert(false); // LCOV_EXCL_LINE
+        return x; // LCOV_EXCL_LINE
+    }
+
+    inline number_unsigned_t remove_sign(number_integer_t x) noexcept
```

Comment:
Could you please add a comment describing the necessity of this function and what the assertion and transformation in the return statement means? Maybe (also) add a link #1708.

Reply:
Done.